src/EditorFeatures/CSharpTest/PullMemberUp/CSharpPullMemberUpTests.cs

```diff
@@ -1,22 +1,55 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.CodeRefactorings;
 using Microsoft.CodeAnalysis.CSharp.CodeRefactorings.PullMemberUp;
 using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeRefactorings;
 using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.CodeAnalysis.CodeRefactorings.PullMemberUp;
 using Xunit;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeRefactorings.PullMemberUp.Dialog;
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis.Test.Utilities.PullMemberUp;
 
 namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.PullMemberUp
 {
     public class CSharpPullMemberUpTests : AbstractCSharpCodeActionTest
     {
         protected override CodeRefactoringProvider CreateCodeRefactoringProvider(Workspace workspace, TestParameters parameters)
-        => new CSharpPullMemberUpCodeRefactoringProvider();
+            => new CSharpPullMemberUpCodeRefactoringProvider(parameters.fixProviderData as IPullMemberUpOptionsService);
```

Comment:
@CyrusNajmabadi  Just tag you here since I remembered in one previous comment you asked will the MEF export the Provider correctly. So it works here since I am create the provider by constructor.